canonical-websites/www.ubuntu.com
# Update vanilla v1.5.1

## Done
Install vanilla-framework v1.5.1. Fix pattern updates in the pages. Remove unnecessary local Scss

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Click through the pages and check there is nothing broken
- Click small and medium screen

## Issue / Card
Fixes https://github.com/canonical-websites/www.ubuntu.com/issues/2210
Fixes https://github.com/canonical-websites/www.ubuntu.com/issues/2211
Fixes https://github.com/canonical-websites/www.ubuntu.com/issues/2212
